src/main/java/org/mafagafogigante/dungeon/schema/rules/JsonRuleFactory.java

```diff
@@ -75,4 +76,8 @@ public static JsonRule makePeriodRule() {
     return new PeriodJsonRule();
   }
 
+  public static JsonRule makeSpecificIdJsonRule(Collection<String> ids) {
```

Comment:
Why not a collection of Id objects? These would certainly be valid, these strings may be anything.